Source/SVONavigation/Private/PathFinding/SVOPathFindingAlgorithmObservers.cpp

```diff
@@ -15,13 +17,15 @@ namespace
         const auto path_points_size = node_addresses.Num() + 1;
 
         path_points.Reset( path_points_size );
+
+        ensureAlways( node_addresses[ 0 ].NodeAddress == params.StartNodeAddress );
         path_points.Emplace( params.StartLocation );

```

Comment:
Should we also add the first node cost to path_point_costs? Since the loop now starts at 1.

Reply:
no because there's no cost for the first location